documentation/feature/advanced_ballistics.md

```diff
@@ -5,14 +5,16 @@ group: feature
 parent: wiki
 ---
 # Overview
-## Sub-feature 1
-Short description of sub-feature 1.
-## Sub-feature 2
-Short description of sub-feature 2.
+## Advanced ballistics simulation
+Advanced ballistics changes above and beyond the ballistics module. Bullet characteristics, barrel length, air temperature, barometric pressure, ammo temperature and air drag are all used to dynamically calculate ballistics.
+
+## Protractor
+Adds a protractor for quick reference when sniping.
 
 # Usage
-Short overview of how to use the feature, e.g. menu options, key bindings, 
-instructions. May not apply to all modules.
+Press `Ctrl+Shift+K` to display the protractor.
```

Comment:
This might be just one of those inconsistencies across documentation. In this case, I think it would be more intuitive if we used the <kbd>keyboard</kbd> element (&lt;kbd&gt;&lt;/kbd&gt;) to mark shortcuts.